src/GuiBase/Viewer/Viewer.hpp

```diff
@@ -90,6 +90,19 @@ class RA_GUIBASE_API Viewer : public WindowQt, public KeyMappingManageable<Viewe
     /// Read-write access to renderer
     Engine::Renderer* getRenderer();
 
+    /** Add a renderer and return its index. Need to be called when catching
+     * \param e : unique_ptr to your own renderer
+     * \return index of the newly added renderer
```

Comment:
If this return an index, why not using `Index` instead of `int` ?

Reply:
This out of scope of this PR, I just reorder declarations to be more consistent. 
We can handle this in a upcoming PR.